protobuf/lib/src/protobuf/coded_buffer_writer.dart

```diff
@@ -220,9 +223,9 @@ class CodedBufferWriter {
     _lastSplicePos = pos;
   }
 
-  /// Add TypedData splice - these bytes would be directly copied into the
-  /// output buffer by [writeTo].
-  void writeRawBytes(TypedData value) {
+  /// Add a [Uint8List] splice, without copying. These bytes will be directly
+  /// copied into the output buffer by [writeTo].
+  void writeRawBytes(Uint8List value) {
```

Comment:
Currently all callers already have a `Uint8List` so this is fine.

But if there were some callers that may need to explicitly create views due to this API, we couldn've made this `void writeRawBytes(Uint8List value, int offset, int length)` (analogous to the `.setRange()` API)

Reply:
Good idea. As you say, currently users always pass a whole `Uint8List` but we can add optional positionals if needed.